src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -79,11 +85,13 @@ public void testFindDefaultDueDateBadParamString() {
         }
         assertTrue("This should have generated an IllegalStateException", caughtError);
```

Comment:
If this class gets converted over to using the JUnit 5 assertion methods, you may want to consider updating the various error-throwing cases to use `assertThrows` assertions instead.